test/end-to-end.js

```diff
@@ -408,13 +408,10 @@ describe('end-to-end', function() {
       .post('/api/Notes')
       .send(sample)
       .expect(200, function(err, res) {
-        if (err) {
-          done(err);
-        } else {
-          expect(res.body).
-            to.have.property('title', 'myTitle');
-          done();
-        };
+        if (err) return done(err);
+        expect(res.body).
+          to.have.property('title', 'myTitle');
```

Comment:
Why not, leave the house cleaner. If you're really nitpicky, separate commit should be good enough.